### PR TITLE
linked both pages

### PR DESCRIPTION
**Pull request by smartcollins**

### CODE REVIEW

File: src/components/Account.jsx [hunk 1]
```diff
@@ -24,10 +23,10 @@ const Account = () => {
     dispatch(incrementByamount())
   }
 
-  useEffect(() => {
-    const timeOut = setTimeout(() => setSignup(true), 3000);
-    return () => timeOut;
-  }, []);
```
Comment (simdi) [inline src/components/Account.jsx:31]: You need to look for another way to solve this problem. Using setTimeout on a web page is not deterministic.

File: src/components/BtmNavbar.jsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { House,SquaresFour,Plus,User } from 'phosphor-react';
 
 const BtmNavbar = ()=>{
     return(
-        <div className='flex items-center justify-around inset-0 fixed bottom-0'>
+        <div className='flex items-center justify-around fixed bottom-0 w-full bg-white py-4 '>
             <House size={30} weight="fill" />
```
Comment (simdi) [inline src/components/BtmNavbar.jsx]: Don't use static values. It will be difficult to maintain.

File: src/components/BtmNavbar.jsx [hunk 3]
```diff
@@ -5,7 +5,7 @@ import { House,SquaresFour,Plus,User } from 'phosphor-react';
...
             <SquaresFour size={30} weight="light" />
```
Comment (simdi) [inline src/components/BtmNavbar.jsx]: Also here. Don't use static values.

**Review by simdi — CHANGES_REQUESTED** [review]

File: src/components/Account.jsx (hunk 1 above)
Comment (smartcollins) [reply src/components/Account.jsx:31]: Ok... thanks Sir

File: src/components/BtmNavbar.jsx (hunk 2 above)
Comment (smartcollins) [reply src/components/BtmNavbar.jsx]: would work on it

File: src/components/BtmNavbar.jsx (hunk 3 above)
Comment (smartcollins) [reply src/components/BtmNavbar.jsx]: Ok sir

File: src/components/BtmNavbar.jsx (hunk 2 above)
Comment (simdi) [reply src/components/BtmNavbar.jsx]: Just to clarify, static value in this case is `size={30}` the 30 in this.

**Pull request merged**